src/ClashOfClans/Models/WarLogEntry.cs

```diff
@@ -10,8 +10,8 @@ public class WarLogEntry
 
         public int? TeamSize { get; set; }
 
-        public WarClan Clan { get; set; }
+        public CurrentWarClan Clan { get; set; }
 
-        public WarClan Opponent { get; set; }
+        public CurrentWarClan Opponent { get; set; }
```

Comment:
Supercell API uses term WarClan for _clan_ and _opponent_, I will not change the class name because it makes future development more difficult and is not in line with SC API.